consensus/msgs.go

```diff
@@ -115,10 +115,10 @@ func MsgToProto(msg Message) (proto.Message, error) {
 			vsb.Votes = *bits
 		}
 
-		pb = vsb
+		pb.Sum = &cmtcons.Message_VoteSetBits{VoteSetBits: vsb}
 
 	default:
-		return nil, ErrConsensusMessageNotRecognized{msg}
+		return pb, ErrConsensusMessageNotRecognized{msg}
```

Comment:
Why this change? (from `nil` to `pb`)
If `pb` is `nil` in this code path, then `nil` is more readable

Reply:
The helper function got changed to return a struct value rather than a type-erased `proto.Message` interface pointer, eliminating a redundant roundtrip of a `Wrap` call and a dynamic downcast.